client/src/legacy/ElementEditor/entwine.js

```diff
@@ -10,7 +10,18 @@ import { destroy } from 'redux-form';
 /**
  * Reset the Apollo and Redux stores holding data relating to elemental inline edit forms
  */
-const resetStores = () => {
+const resetStores = (numAttempts = 0) => {
+  // If there are queries waiting to be performed, resetting the cache store will cancel them.
+  // Make sure all queries are complete before resetting stores.
+  // Only try 3 times - after that clearing the store is more likely to cause noticable issues
+  // than resolve them.
+  if (window.ss.apolloClient.queryManager.fetchQueryRejectFns.size > 0 && numAttempts < 3) {
+    setTimeout(() => {
+      resetStores(numAttempts + 1);
+    }, 200);
```

Comment:
```suggestion
    }, 500);
```

Reply:
Could we increase to half a second. With 200ms I still see Error in browser console.

Reply:
I have this error in console after 3rd attempt when I save or publish page and ElementalArea is refreshing. I tested it a few times with different time value and it fails on 200, 300 and 400ms, but not on 500ms.
If you say, that increasing time could be a cause of another issue, so we can add this code, but it doesn't solve this issue.

Reply:
I don't think increasing the timestep is an option for the reasons I've mentioned.
If this is _always_ failing for you with the 200ms timestep then this PR is not a valid improvement and should be closed, and the ticket put back in the backlog.

Reply:
I'll close it before we can find a better solution.